qemu/tests/win_virtio_driver_install_by_installer.py

```diff
@@ -40,6 +41,9 @@ def run(test, params, env):
     vm = env.get_vm(params["main_vm"])
     session = vm.wait_for_login()
 
+    if params.get("installer_chk_cmd"):
+        win_installer_version_test(session, test, params)
+        return
```

Comment:
@xiagao Sorry, after a second review, I think here it does not make sense to have it in this script, it has nothing to do with the whole script. I think we should have a separate script for this case. I don't think the function win_installer_version_test will be reused anywhere else, so it is also not make sense to have it in `provider`, the whole function should be an independent script.